roles/tpa_single_node/meta/argument_specs.yml

```diff
@@ -0,0 +1,398 @@
+      tpa_single_node_storage_vexination_bucket:
+        description: "V11y storage bucket name."
+        type: "str"
+        version_added: "1.2.0"
+        default: "vexination-default"
+      tpa_single_node_storage_region:
+        description: "AWS S3 Storage region"
+        type: "str"
+        version_added: "1.2.0"
+        default: "eu-west-1"
+      tpa_single_node_storage_endpoint:
+        description: "Minio storage endpoint if used instead of S3"
+        type: "str"
+        version_added: "1.2.0"
+        default: "eu-west-1"
+      tpa_single_node_event_bus_type:
+        description: "Kafka or SQS"
+        type: "str"
+        version_added: "1.2.0"
+        default: "kafka"
+      tpa_single_node_event_access_key_id:
+        description: "Kafka Username or AWS SQS Access Key ID, readed from TPA_EVENT_ACCESS_KEY_ID env var"
+        type: "str"
+        required: true
+        version_added: "1.2.0"
+      tpa_single_node_event_secret_access_key:
+        description: "Kafka password or AWS SQS Secret Access Key, readed from TPA_EVENT_SECRET_ACCESS_KEY env var"
+        type: "str"
+        required: true
+        version_added: "1.2.0"
+      tpa_single_node_bombastic_topic_failed:
+        description: "Bombastic Events topic failed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "bombastic-failed-default"
+      tpa_single_node_bombastic_topic_indexed:
+        description: "Bombastic Events topic indexed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "bombastic-indexed-default"
+      tpa_single_node_bombastic_topic_stored:
+        description: "Bombastic Events topic stored"
+        type: "str"
+        version_added: "1.2.0"
+        default: "bombastic-stored-default"
+      tpa_single_node_vexination_topic_failed:
+        description: "Vexination Events topic failed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "vexination-failed-default"
+      tpa_single_node_vexination_topic_indexed:
+        description: "Vexination Events topic indexed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "vexination-indexed-default"
+      tpa_single_node_vexination_topic_stored:
+        description: "Vexination Events topic stored"
+        type: "str"
+        version_added: "1.2.0"
+        default: "vexination-stored-default"
+      tpa_single_node_v11y_topic_failed:
+        description: "V11y Events topic failed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "vv1y-failed-default"
+      tpa_single_node_v11y_topic_indexed:
+        description: "V11y Events topic indexed"
+        type: "str"
+        version_added: "1.2.0"
+        default: "v11y-indexed-default"
+      tpa_single_node_v11y_topic_stored:
+        description: "V11y Events topic stored"
+        type: "str"
+        version_added: "1.2.0"
+        default: "v11y-stored-default"
+      tpa_single_node_kafka_bootstrap_servers:
```

Comment:
I hope the kafka fields are only required if the event type is Kafka or else it is optional. Is there a way to add a condition for this his here?